Orange/widgets/io.py

```diff
@@ -35,8 +35,16 @@ def _export(self, exporter, filename):
     @classmethod
     def write_image(cls, filename, scene):
         try:
+            scene = scene.scene()
+            scenerect = scene.sceneRect()   #preserve scene bounding rectangle
+            viewrect = scene.views()[0].sceneRect()
+            scene.setSceneRect(viewrect)
+            backgroundbrush = scene.backgroundBrush()  #preserve scene background brush
+            scene.setBackroundBrush(Qt.White)
```

Comment:
thank you for pointing that out, apparently  it was still working because the scene was already updated before the error occurred and hence we were getting the expected results. Fixed it...